src/encode.c

```diff
@@ -804,7 +804,13 @@ PyImaging_LibTiffEncoderNew(PyObject* self, PyObject* args)
     PyObject *dir;
     PyObject *key, *value;
     Py_ssize_t pos = 0;
-    int status;
+    int key_int, status, core, i;
+    // This list also exists in TiffTags.py
+    const int tags[32] = {
```

Comment:
Do we need to specify the size of the array?
```suggestion
    const int tags[] = {
```